cves/CVE-2017-7675.yml

```diff
@@ -204,4 +236,13 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: |
+    Based on the comments surrounding the original implementation, as well as
+    the brief history of commits revolving around the chunk of code containing
+    the vulnerability and the lack of targeted unit tests; this vulnerability
+    was most certainly caused by a lack of proper risk assessment. 
+
+    With a comment like "This is almost certainly wrong and needs to be decoded",
```

Comment:
Was that comment actually in the code or commit history? Was there any explanation provided for why it was decided that the code should be accepted?

Reply:
Will clarify! The vulnerable code ended up replacing some code with the comment "TODO: This is almost certainly wrong and needs to be decoded".